ecosystem/sep-0006.md

```diff
@@ -1078,6 +1082,7 @@ The response should be a JSON object like:
       "authentication_required": true,
       "min_amount": 0.1,
       "max_amount": 1000,
+      "funding_method": ["SEPA", "SWIFT", "cash"],
       "fields": {
```

Comment:
Lets remove these `fields` objects from examples given that the object is deprecated